components/form-builder/panel/PanelActions.tsx

```diff
@@ -74,15 +94,25 @@ export const PanelActions = ({
   children: React.ReactNode;
 }) => {
   const { t } = useTranslation("form-builder");
-  const { remove, moveUp, moveDown, add, duplicateElement } = useTemplateStore();
+  const { remove, moveUp, moveDown, add, duplicateElement, form } = useTemplateStore();
+  const isLastItem = item.index === form.elements.length - 1;
+  const isFirstItem = item.index === 0;
 
   return (
     <Actions className="panel-actions">
       <UpDown>
-        <PanelButton icon={<ChevronUp />} onClick={() => moveUp(item.index)}>
+        <PanelButton
+          className={isFirstItem ? "disabled" : ""}
+          icon={<ChevronUp />}
+          onClick={isFirstItem ? undefined : () => moveUp(item.index)}
+        >
```

Comment:
Better to add [the `disabled` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-disabled) rather than a class. Then you can just keep the `onClick` except it shouldn't work.

And your css can be like `button[disabled] { ... }`